src/fsharp/FSharp.Core.nuget/FSharp.Core.4.2.xxx.nuspec

```diff
@@ -1,20 +1,36 @@
 <?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
     <metadata>
-        <id>Microsoft.FSharp.Core.netcore</id>
-        <description>
-            .NET Core compatible version of the fsharp core library FSharp.Core.dll
-            Supported Platforms: - .NET Core (netstandard1.6)
-        </description>
+        <!-- rename to FSharp.Core when done -->
+        <id>FSharp.Core</id>
+        <title>FSharp.Core for F# 4.1</title>
+        <summary>FSharp.Core for F# 4.1</summary>
+        <description>FSharp.Core for F# 4.1
+
+          FSharp.Core redistributables from Visual F# 4.1.0
+
+          .NET 2.0
+          .NET 4.0
+          .NET Portable Profile 7 (portable-net45+netcore45)
+          .NET Portable Profile 47 (portable-net45+sl5+netcore45)
+          .NET Portable Profile 78 (portable-net45+netcore45+wp8)
+          .NET Portable Profile 259 (portable-net45+netcore45+wpa81+wp8)
+          .NET Standard 1.6 (netstandard1.6)
+
+          MonoAndroid, MonoTouch, Xamarin.iOS (portable-net45+monoandroid10+monotouch10+xamarinios10)
+          XamarinMac mobile (xamarinmac20)</description>
         <language>en-US</language>
-        <requireLicenseAcceptance>true</requireLicenseAcceptance>
         <version>$version$</version>
         <authors>$authors$</authors>
         <licenseUrl>$licenseUrl$</licenseUrl>
         <projectUrl>$projectUrl$</projectUrl>
         <tags>$tags$</tags>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <dependencies>
             <group>
+                <dependency id="System.ValueTuple" version="4.3.0" />
```

Comment:
Oh ... I see, at some point in my head I got the two packages mixed up.  4.1.xxx has a dependence on ValueTuple, because the existing 4.1.17 package does, it's just I stupidly put that in 4.2.xx along with the comments.